examples/jsm/loaders/VRMLLoader.js

```diff
@@ -34,6 +34,7 @@ import {
 	Vector3,
 	VertexColors
 } from "../../../build/three.module.js";
+import chevrotain from "../libs/chevrotain.min.js";
 
 /* global chevrotain */
```

Comment:
I think you should be able to remove this line now, as well.

Reply:
We can do this in a separate PR. Lines line this exists in multiple files.

Reply:
`examples/jsm/loaders/VRMLLoader.js` is generated from `examples/js/loaders/VRMLLoader.js` where the comment is still needed. 

We could adapt `utils/modularize.js` to remove the global. Adapting the global line to remove imported globals would make the code unnecessarily complex. We could remove the global comment completely once all modules have their dependencies correctly imported.

Reply:
Oh yes I forgot that file is generated :X

Feel free to ignore :) Modifying the script seems overkill for a non functional change. I say keep as is.